apm/src/app/user/user.service.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from '@angular/core';
+import { IUser } from './user';
+
+@Injectable({
+  providedIn: 'root'
+})
+
+export class UserService {
+
+  public currentUser!: IUser | null;
+    
+  login(userName: string, password: string) {
+    this.currentUser = {
```

Comment:
.

Reply:
I didn't get what you meant here. Could you please brief on this?

Reply:
nothing, I did not know how to delete the comment, so simply put it as `.`